src/api-reference/expense/allocations/expense-allocations.markdown

```diff
@@ -3,20 +3,18 @@ title: Allocations
 layout: reference
 ---
 
-# Allocations
-
 This resource can be used to retrieve information about the allocations that are associated with an entry in an expense report. v3.0
 
 
-* [Retrieve all allocations per entry or report](#get)
-* [Retrieve a single allocation by ID](#getID)
+* [Retrieve All Allocations Per Entry or Report](#get)
+* [Retrieve a Single Allocation by ID](#getID)
 * [Schema](#schema)
-* [Try it out](#try-it-out)
+* [Try It Out](#try-it-out)
```

Comment:
Delete this section (both the TOC link and the section below), it doesn't work the way they intended.